include/hyprcursor/hyprcursor.hpp

```diff
@@ -47,6 +47,20 @@ namespace Hyprcursor {
         eHyprcursorDataType               type        = HC_DATA_PNG;
     };
 
+    /*!
+        struct for cursor manager options
+    */
+    struct SManagerOptions {
```

Comment:
needs a library-side explicit ctor to avoid ABI breakages

Reply:
this aint solved

Reply:
why a constructor for SManagerOptions?
maybe Im missing something

Reply:
because by doing `= something;` you're creating an implicit ctor on the client's side - so when you get a client compiled against ver 1.1 which has 2 options in managerOptions, it initializes 2, but if library is 1.2 and adds a third, library will read garbage.

Reply:
`explicit SManagerOptions();` and in cpp `SManagerOptions::SManagerOptions() { a = 0; b = 2; }`

Reply:
ok makes sense